.github/run_kuttl_tests.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -ex
+
+# Install kuttl
+if [[ ! -f /usr/local/bin/kubectl-kuttl ]]; then
+    wget -O /usr/local/bin/kubectl-kuttl https://github.com/kudobuilder/kuttl/releases/download/v0.9.0/kubectl-kuttl_0.9.0_linux_x86_64
+    chmod +x /usr/local/bin/kubectl-kuttl
+fi
+
+
+OPERATOR=$(echo ${1} | cut -d '-' -f1)
```

Comment:
If **ovn-operator** renames to **ovn-northd-operator**, it will fail to.
Checking on install_yamls, ansibleee expects:
ANSIBLEEE_IMG, ANSIBLEEE_REPO.
So we may need to also cut out "openstack-" from operator's name.